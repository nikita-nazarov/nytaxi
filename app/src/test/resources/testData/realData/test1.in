1230772277000000,1588371493000000