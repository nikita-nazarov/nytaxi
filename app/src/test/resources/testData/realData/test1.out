0,2.58444241871771
1,2.60075903377295
2,2.81188559649575
3,2.69510432681748
4,2.46819435736677
5,2.5364907367514
6,2.72982608695652
7,0.0